Editorial pass for minor style guide issues and clarity (#1)
StakeNow/namespaces: tezos/caip2.md

### Chain ID alias

The Tezos community recognizes the different chains according to human readable names, which are called the "[Networks][]".
The Octez RPC, for example, allows you to connect to three predefined networks by alias:

```bash
# mainnet (default)
# sandbox
# ghostnet

> ./octez-node run --data-dir ~/tezos-ghostnet --network ghostnet
```

> There is currently no algorithm to connecting `chain ID`s to `network`s in the Octez reference implementation of Tezos.

Author: ```suggestion
There is currently no algorithm for connecting to a network in the Octez reference implementation of Tezos by that network's CAIP-2 chainId, nor a way to query a node directly for a chainId.
```

| Alias          | Chain ID                         |
| -------------- | -------------------------------- |

> | tezos mainnet  | tezos:NetXdQprcVkpaWU            |

Author: ```suggestion
| tezos:mainnet  | tezos:NetXdQprcVkpaWU            |
```

> | tezos ghostnet | tezos:NetXnHfVqm9iesp            |

Author: ```suggestion
| tezos:ghostnet | tezos:NetXnHfVqm9iesp            |
```
